libraries/app/application.cpp

```diff
@@ -404,6 +405,9 @@ namespace detail {
             _force_validate = true;
          }
 
+         if( _options->count("enable-subscribe-to-all") )
+            _app_options.enable_subscribe_to_all = _options->at("enable-subscribe-to-all").as<bool>();
```

Comment:
@pmconrad perhaps this `as` call need to be revised along with other changes in #714?

Reply:
This as() is not a variant::as(), it's from boost::program_options. I don't think we need to sanitize that.